app/src/main/java/org/schabi/newpipe/util/AnimationUtils.java

```diff
@@ -43,6 +46,26 @@
         SLIDE_AND_ALPHA, LIGHT_SLIDE_AND_ALPHA
     }
 
+    /**
+     * Animate the progress bar's progress
+     *
+     * @param progressBar progress bar that will be animated
+     * @param to          final progress value after animation
+     * @param duration    how long the animation will take, in milliseconds
+     */
+    public static void animateProgressBar(ProgressBar progressBar, int to, long duration) {
```

Comment:
Put this function below the others, but on top of the `Internals` block.

Reply:
OK, I'll do